src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2529,23 +2529,81 @@ public static ArgumentSyntax Argument(ExpressionSyntax expression)
         /// <summary>Creates a new ArgumentSyntax instance.</summary>
         public static ArgumentSyntax Argument(NameColonSyntax nameColon, SyntaxToken outKeyword, VariableDeclarationSyntax declaration)
         {
-            if (outKeyword.Kind() != SyntaxKind.OutKeyword)
+            return Argument(nameColon, outKeyword, (CSharpSyntaxNode)declaration);
```

Comment:
I don't believe either factory takes CSharpSyntaxNode expressionOrDeclaration. What am I missing?

---
In reply to: [65972883](https://github.com/dotnet/roslyn/pull/11765#discussion_r65972883) [](ancestors = 65972883)